psqlgraph/node.py

```diff
@@ -81,7 +59,6 @@ def __declare_last__(cls):
                     )
                     setattr(cls, name_in, edge_in)
                     cls._edges_in.append(name_in)
-                    dst_ids.append(scls.dst_id)
```

Comment:
?

Reply:
Having a single question mark does not really tell me what the question/concern/comment is/means? If the question is why the line was removed, its a part of a larger refactor.

Reply:
I made this change, `dst_ids` were not used at all, so I got rid of it.

Reply:
thanks for clarifying 😄